README.md

```diff
@@ -1,6 +1,7 @@
 # phyx.js
 
 [![Build Status](https://github.com/phyloref/phyx.js/workflows/Build%20and%20Test/badge.svg)](https://github.com/phyloref/phyx.js/actions?query=workflow%3A%22Build+and+Test%22)
+[![DOI](https://zenodo.org/badge/166467036.svg)](https://zenodo.org/badge/latestdoi/166467036)
```

Comment:
Is there a reason we shouldn't just be using the DOI for "the latest version" on Zenodo, as included in the CITATION.cff file?

Reply:
Fixed in 16324ef.